src/Orleans.Core/Configuration/CollectionAgeLimitAttribute.cs

```diff
@@ -8,6 +8,25 @@ namespace Orleans.Configuration
     [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
     public class CollectionAgeLimitAttribute : Attribute
     {
-        public double Minutes { get; set; }
+        public const int DefaultAgeMinutes = 10;
```

Comment:
Why 10 minutes? The default today is 2 hours.

Reply:
Can move the ```GrainCollectionOptions``` from ```Orleans.Runtime``` project to ```Orleans.Core``` project ?
 if so I can use the ```GrainCollectionOptions.DEFAULT_COLLECTION_AGE_LIMIT```

> Why 10 minutes? The default today is 2 hours.

Reply:
I don't think moving `GrainCollectionOptions` to `Orleans.Core` makes a lot of sense. But we can move `DEFAULT_COLLECTION_AGE_LIMIT` from `GrainCollectionOptions` to `CollectionAgeLimitAttribute`. I submitted https://github.com/bluexo/orleans/pull/1 with the change.